transform/mattermost-analytics/models/staging/mm_calls_test_go/_mm_calls_test_go__sources.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+
+sources:
+  - name: mm_calls_test_go
```

Comment:
maybe add a note - why is this model being called "test" - to be fixed in the future.

Reply:
Updated note, will look into renaming if possible.

Reply:
After this PR has been merged, we can rename the source and join the two schemas. It should require creating a copy of the two staging models and a new model that unions old and new staging models.